### PR TITLE
New Rule: Attachment: RTF file with suspicious link

### PR DESCRIPTION
**Pull request by morriscode**
# Description

New Rule detecting RTF File attachments (direct or within archive) with suspicious links
Exclusions made for image links

# Associated samples

[Sample](https://platform.sublimesecurity.com/messages/7aefecc7df80eed02d0b3517128faef7b34195ea55e8b57fa1eba707988162b8)

### CODE REVIEW

**Comment by morriscode** [comment]
The newer test rules dashboard is confused by this rule getting a name change. The old one shows one hit. 
In my testing however this is knocking down hundreds in sharedEML's and is probably tight enough with the scope on RTF's to go out.

File: detection-rules/attachment_rtf_file_with_suspicious_link.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                            or (
+                              .domain.root_domain in $free_file_hosts
+                              or .domain.root_domain in $free_file_hosts
+                              or .domain.root_domain in $free_subdomain_hosts
+                              or .domain.root_domain in $url_shorteners
+                            )
+                            // or the url contains the recipient email and the root_domain is not in tranco
+                            or (
+                              any(recipients.to,
+                                  strings.icontains(..url, .email.email)
```
Comment (zoomequipd) [inline detection-rules/attachment_rtf_file_with_suspicious_link.yml:35]: Do you want `..url` here or `.url`?

**Review by zoomequipd — CHANGES_REQUESTED** [review]

File: detection-rules/attachment_rtf_file_with_suspicious_link.yml (hunk 1 above)
Comment (zoomequipd) [reply detection-rules/attachment_rtf_file_with_suspicious_link.yml:35]: `..url` is correct

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**